Source/Modules/csharp.cxx

```diff
@@ -1456,6 +1552,15 @@ class CSHARP:public Language {
     Swig_save("constantWrapper", n, "value", NIL);
     Swig_save("constantWrapper", n, "tmap:ctype:out", "tmap:imtype:out", "tmap:cstype:out", "tmap:out:null", "tmap:imtype:outattributes", "tmap:cstype:outattributes", NIL);
 
+    //translate and write comment if flagged
```

Comment:
Sorry, could you please explain what do you mean by "if flagged" here?

Reply:
this is a copy from the java module, no idea what it actually means, probably "is enabled?", not very useful in that case

Reply:
Blind copy from the Java module, probably means "if enabled", removed now.